projects/wave-core-new/src/lib/wave-core.module.ts

```diff
@@ -273,6 +276,7 @@ const WAVE_COMPONENTS = [
         FlexLayoutModule,
         FormsModule,
         HttpClientModule,
+        NgxMatSelectSearchModule,
```

Comment:
If it is now here, maybe check if we can remove this from the ebv app

Reply:
it's still needed for the country selector